libdnf/module/module_metadata.cpp

```diff
@@ -167,4 +167,43 @@ std::pair<std::vector<ModuleItem *>, std::vector<ModuleItem *>> ModuleMetadata::
 }
 
 
+std::map<std::string, std::string> ModuleMetadata::get_default_streams() {
+    std::map<std::string, std::string> default_streams;
+    if (!resulting_module_index) {
```

Comment:
Please could you add a comment why resulting_module_index can be absent or something else?

Reply:
I guess that it should be only absent if there are no modules. But I guess that it can also happened that module defaults were not resolved.